<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDisplay</class>
 <widget class="QWidget" name="FormDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>550</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../usr/share/kademar/utils/cadi/img/bg/cadi.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="pages">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="software">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>471</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Display Config</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>You can change your display settings.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="l_logo_blth">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../usr/share/kademar/utils/cadi/img/display.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Display Resolution</string>
     </property>
     <widget class="QLabel" name="l_resolution">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>800x600</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="sldr_resol">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Color Depth</string>
     </property>
     <widget class="QComboBox" name="cb_depth">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>24 bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 bit</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cb_forceResol">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Force
Resolution</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>General Font Size (DPI)</string>
     </property>
     <widget class="QComboBox" name="cb_dpi">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Too Large</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Too Small</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="l_logo_blth_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>225</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../usr/share/kademar/utils/cadi/img/color.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_logo_blth_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>335</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../usr/share/kademar/utils/cadi/img/font.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Resolution Confirmation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>481</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Resolution has been changed.
If you look this message correctly, press OK.

If you have forced a resolution , you may need to restart the session.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>If you don't confirm it, resolution will be restored on:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="l_seconds">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seconds</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="l_countdown">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="b_ok">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>390</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_restore">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/reload.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/reload.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/kademar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sortir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/enrera.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_SaX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
